xbmc/platform/darwin/tvos/input/LibInputTouch.mm

```diff
@@ -364,283 +362,189 @@ - (IBAction)IRRemoteRightArrowPressed:(UIGestureRecognizer*)sender
 
 - (IBAction)tapUpArrowPressed:(UIGestureRecognizer*)sender
 {
-  if (!g_xbmcController.inputHandler.inputRemote.remoteIdleState)
+  CLog::Log(LOGDEBUG, "Input: Siri remote tap up (id: 1)");
+  if (!g_xbmcController.inputHandler.inputRemote.siriRemoteIdleState)
     [g_xbmcController.inputHandler sendButtonPressed:1];
 
-  [g_xbmcController.inputHandler.inputRemote startRemoteTimer];
+  [g_xbmcController.inputHandler.inputRemote startSiriRemoteIdleTimer];
 }
 
 - (IBAction)tapDownArrowPressed:(UIGestureRecognizer*)sender
 {
-  if (!g_xbmcController.inputHandler.inputRemote.remoteIdleState)
+  CLog::Log(LOGDEBUG, "Input: Siri remote tap down (id: 2)");
+  if (!g_xbmcController.inputHandler.inputRemote.siriRemoteIdleState)
     [g_xbmcController.inputHandler sendButtonPressed:2];
 
-  [g_xbmcController.inputHandler.inputRemote startRemoteTimer];
+  [g_xbmcController.inputHandler.inputRemote startSiriRemoteIdleTimer];
 }
 
 - (IBAction)tapLeftArrowPressed:(UIGestureRecognizer*)sender
 {
-  if (!g_xbmcController.inputHandler.inputRemote.remoteIdleState)
+  CLog::Log(LOGDEBUG, "Input: Siri remote tap left (id: 3)");
+  if (!g_xbmcController.inputHandler.inputRemote.siriRemoteIdleState)
     [g_xbmcController.inputHandler sendButtonPressed:3];
 
-  [g_xbmcController.inputHandler.inputRemote startRemoteTimer];
+  [g_xbmcController.inputHandler.inputRemote startSiriRemoteIdleTimer];
 }
 
 - (IBAction)tapRightArrowPressed:(UIGestureRecognizer*)sender
 {
-  if (!g_xbmcController.inputHandler.inputRemote.remoteIdleState)
+  CLog::Log(LOGDEBUG, "Input: Siri remote tap right (id: 4)");
+  if (!g_xbmcController.inputHandler.inputRemote.siriRemoteIdleState)
     [g_xbmcController.inputHandler sendButtonPressed:4];
 
-  [g_xbmcController.inputHandler.inputRemote startRemoteTimer];
+  [g_xbmcController.inputHandler.inputRemote startSiriRemoteIdleTimer];
 }
 
 #pragma mark - Pan
 
 - (IBAction)handlePan:(UIPanGestureRecognizer*)sender
 {
-  if (g_xbmcController.inputHandler.inputRemote.remoteIdleState)
+  if (g_xbmcController.inputHandler.inputRemote.siriRemoteIdleState)
+  {
+    [g_xbmcController.inputHandler.inputRemote startSiriRemoteIdleTimer];
     return;
+  }
 
-  if (!g_xbmcController.appAlive) //NO GESTURES BEFORE WE ARE UP AND RUNNING
-    return;
+  CGPoint translation = [sender translationInView:sender.view];
+  CGPoint velocity = [sender velocityInView:sender.view];
+  UIPanGestureRecognizerDirection direction = [self getPanDirection:velocity];
+  //  CLog::Log(LOGDEBUG, "Input: pan (point: (%f,%f), velocity: (%f,%f))", translation.x, translation.y, velocity.x, velocity.y);
```

Comment:
Please update to the current log standards using `{}`.

For more information please see the [libfmt documentation](https://fmt.dev/latest/syntax.html)